README.md

```diff
@@ -4,37 +4,61 @@
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
-## Running the website locally using Hugo
+# Using this repository
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run the website locally using Hugo, or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 
-Before building the site, clone the Kubernetes website repository:
+## Prerequisites
 
-```bash
+To use this repository, you need the following installed locally:
+
+- [yarn](https://yarnpkg.com/)
+- [npm](https://www.npmjs.com/)
+- [Go](https://golang.org/)
+- [Hugo](https://gohugo.io/)
+- A container runtime, like [Docker](https://www.docker.com/).
+
+Before you start, install the dependencies. Clone the repository and navigate to the directory:
+
+```
 git clone https://github.com/kubernetes/website.git
 cd website
-git submodule update --init --recursive --depth 1
 ```
 
-**Note:**  The Kubernetes website deploys the [Docsy Hugo theme](https://github.com/google/docsy#readme).
-If you have not updated your website repository, the `website/themes/docsy` directory is empty. The site cannot build
-without a local copy of the theme.
+The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
```

Comment:
nit: Change from "strongly recommend pulling ..."
The site does not build (locally Hugo) without the theme. The build generates error messages that are not indicative of the actual problem.

Reply:
If you're using the `Makefile` and didn't pull the submodules, the first line of the failing output is (now):
```
WARNING Submodule not initialized: themes/docsy
```

which might be enough of a hint to look at the README(?)

Reply:
It _is_ a good idea to require pulling the submodules; you need them.

Reply:
Rather, I'd say it is a requirement for building locally with Hugo.